RNDeviceInfo.podspec

```diff
@@ -13,10 +15,13 @@ Pod::Spec.new do |s|
   s.platforms     = { :ios => "9.0", :visionos => "1.0", :tvos => "10.0"}
 
   s.source       = { :git => "https://github.com/react-native-device-info/react-native-device-info.git", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.source_files = "ios/**/*.{h,m,mm,swift}"
   s.resource_bundles = {
     'RNDeviceInfoPrivacyInfo' => ['ios/PrivacyInfo.xcprivacy'],
   }
 
+  s.frameworks = 'CoreTelephony', 'WebKit', 'LocalAuthentication', 'DeviceCheck'
```

Comment:
Why do we need them now?

Reply:
It won't compile without them. I'll try removing it again

Reply:
<img width="354" alt="Screenshot 2024-07-01 at 09 39 06" src="https://github.com/react-native-device-info/react-native-device-info/assets/30924086/0e070259-8ac3-40bf-a341-32dd88db058e">

Reply:
Not sure why it's happening because `$(inherited)` is added to the linker flags